governance/sig-governance.md

```diff
@@ -0,0 +1,81 @@
+- MAY participate in decision making of the SIG
+- Members are tracked in `membership.md` in the specific SIG (Contributors are listed in the aggregated contributor list for maintainence convenience)
+
+### Tech Lead
+
+  - **Number**: 2-3. Initially assigned by Maintainers
+  - **Requirement**:
+    - MUST be a committer from the specific SIG
+    - MUST have demonstrated deep understanding of the SIG project
+  - **Term**: 1 year
+  - **Election:**
+      - Nominated by the Tech Leads or self-nominate
+      - Gained supermajority votes of Maintainers
+      - Done through PR to update the `membership.md` file in the SIG
+- **Responsibilities**
+    - Creating new projects under the SIG
+    - Triaging issues/tasks in the SIG
+    - Mentoring members of the SIG
+    - Organizing discussions and drawing conclusions for decision making within the SIG
+    - Ensuring health and sustained development of the SIG and grwoth of SIG members
+    - Making roadmap for the SIG
+    - Organizing and participating in regular meetings
+
+## Member Promotion Mechanism
+
+Members of the SIG can be promoted to a higher role based on the rules defined in [TiDB Developer Group](/architecture/README.md#tidb-developer-group). 
+
+## Member Exit Mechanism
+
+There are two exit situations:
+
+- Members MAY decide to step down at anytime for personal reasons by opening the PR to modify the `member-list.md`in the Specific SIG. The PR can be merged once it gets two approvals from the higher role.
+
+- Members of a role SHOULD be removed from the SIG if they have not communicated a leave of absence and either cannot be reached for more than 1 month or are not fulfilling their documented responsibilities for more than 1 month. Tech Leads will assess the communication and contribution status monthly to update the membership list. Each change (removal), depending on the role, requires the following:
+
+    - For Committer or Tech Leads, a supermajority votes from Maintainers are required
+    - For Reviewers or lower roles, two approvals from the immediate higher role is required
+
+Once the change is approved, the corresponding profile will be moved to the emeritus role list.
+
+## SIG Lifecycle
+
+### Creation
+
+1. Copy the [SIG Charter Template](SIG-CHARTER-TEMPLATE.md).
+2. Modify the content in the template that needs to be defined for the specific SIG
+3. Initiate a PR in [special-interest-groups](https://github.com/pingcap/community/tree/master/special-interest-groups) with the SIG Charter, and propose the SIG Charter. This must be approved by the supermajority of the PMC members.
+4. Create a public and private Slack Channel in tikv-wg.slack.com to discuss the SIG operation related matters.
+5. Create a Zoom room for regular meetings, as well as other online discussions.
+6. Announce the establishment of a new SIG in the TiKV community.
```

Comment:
SIG creator also needs to create a public google doc to summary the regular SIG work report and record the regular meeting notes.

Reply:
Added